sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/csv/CSVBenchmarks.scala

```diff
@@ -137,22 +124,15 @@ object CSVBenchmarks extends SQLHelper {
         ds.count()
       }
 
-      /*
-      Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
-
-      Count a dataset with 10 columns:      Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
-      ---------------------------------------------------------------------------------------------
-      Select 10 columns + count()              12598 / 12740          0.8        1259.8       1.0X
-      Select 1 column + count()                  7960 / 8175          1.3         796.0       1.6X
-      count()                                    2332 / 2386          4.3         233.2       5.4X
-      */
       benchmark.run()
     }
   }
 
-  def main(args: Array[String]): Unit = {
-    quotedValuesBenchmark(rowsNum = 50 * 1000, numIters = 3)
-    multiColumnsBenchmark(rowsNum = 1000 * 1000)
-    countBenchmark(10 * 1000 * 1000)
+  override def runBenchmarkSuite(): Unit = {
```

Comment:
#22872 has updated `runBenchmarkSuite`'s signature.
```suggestion
  override def runBenchmarkSuite(mainArgs: Array[String]): Unit = {
```

Reply:
+1 for @yucai 's comment.